lib/lytics/index.js

```diff
@@ -62,7 +63,10 @@ Lytics.prototype.loaded = function(){
  */
 
 Lytics.prototype.page = function(page){
-  window.jstag.send(page.properties());
+  var props = page.properties();
+  props._e = props.name;
+  delete props.name;
```

Comment:
we're trying to avoid `delete` and to avoid reliance on specific key casing (ie. `.name`).

```
var props = foldl(function(ret, val, key){  }, {}, page.properties());
```